docs/networking/networking.md

```diff
@@ -63,13 +63,15 @@ To exclude nodes from being used by ServiceLB, add the following label to the no
 svccontroller.k3s.cattle.io/enablelb
 ```
 
-To select a particular subset of nodes to host pods for a LoadBalancer, set matching annotation values on the Nodes and Service. For example:
+To select a particular subset of nodes to host pods for a LoadBalancer, set matching label values on the Nodes and Service. For example:
 
 1. Label Node A and Node B with `svccontroller.k3s.cattle.io/lbpool=pool1`
 2. Label Node C and Node D with `svccontroller.k3s.cattle.io/lbpool=pool2`
 3. Create one LoadBalancer Service on port 443 with label `svccontroller.k3s.cattle.io/lbpool=pool1`. The DaemonSet for this service only deploy Pods to Node A and Node B.
 4. Create another LoadBalancer Service on port 443 with label `svccontroller.k3s.cattle.io/lbpool=pool2`. The DaemonSet will only deploy Pods to Node C and Node D.
 
+The svccontroller.k3s.cattle.io/enablelb label must also be set on the nodes otherwise the pool label will be ignored.
```

Comment:
One nit on the text here, but this is somewhat tricky to explain simply -  If you're using the `enablelb` label on other nodes in the cluster, it must **also** be used on the node that the `lbpool` label is on, but if you're not already using it on other nodes, it does't need to be set.
```suggestion
The `svccontroller.k3s.cattle.io/enablelb` label must also be set on the nodes otherwise the pool label will be ignored.
```